pkg/operator/starter.go

```diff
@@ -522,6 +525,11 @@ func prepareOauthAPIServerOperator(ctx context.Context, controllerContext *contr
 	}
 	apiregistrationInformers := apiregistrationinformers.NewSharedInformerFactory(apiregistrationv1Client, 10*time.Minute)
 
+	operatorClient, err := operatorconfigclient.NewForConfig(controllerContext.KubeConfig)
```

Comment:
renaming `operatorClient` to `typedOperatorClient` would make reading the code slightly more pleasant.

Reply:
I think that in line [124](https://github.com/openshift/cluster-authentication-operator/blob/3ac53812cf416ee33f873497ba25bc13a6dcda28/pkg/operator/starter.go#L124) we already created the typed client. Should we reuse it instead ?

Reply:
> I think that in line [124](https://github.com/openshift/cluster-authentication-operator/blob/3ac53812cf416ee33f873497ba25bc13a6dcda28/pkg/operator/starter.go#L124) we already created the typed client. Should we reuse it instead ?

Different function used to drive informers on a bad resync cycle.  One of the refactors will remove the one from line 124.